.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,92 @@
+name: Bug Report
+description: Create a report about an error to help us improve
+
+type: bug
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to contribute to icepyx by filling out this bug report! We appreciate
+        your help in improving icepyx and making data more accessible.
+
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: Describe the bug
+      description: |
+        A clear and concise description of what the problem is. In 1-3 sentences, summarize what you were
+        trying to accomplish and what happened instead.
+      placeholder: |
+        Example: While attempting to download 2 granules using `region.download_granules()` I got an error
+        informing me that I needed to login, but I had already provided my login information.
+
+        Example: While searching for data using `ipx.Query()` using the `spatial_extent` filter but upon
+        opening my files I see that some of the granules are located outside the specified spatial extent.
+    validations:
+      required: true
+
+  - type: textarea
+    id: expected
+    attributes:
+      label: Expected behavior
+      description: |
+        A clear and concise description of what you expected to happen.
+      placeholder: |
+        I expected 12 granules. The granules ids I expected were ...
+
+  - type: textarea
+    id: sample-code
+    attributes:
+      label: Reproducible Example
+      description: |
```

```suggestion
      description: >
```
I think the ">" instead of the "|" will make the rendered text wrap neatly instead of matching linebreaks exactly.